img/m/microworkers.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" width="16" height="16"><image xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAIGNIUk0AAHomAACAhAAA+gAAAIDo AAB1MAAA6mAAADqYAAAXcJy6UTwAAACiUExURf////CiouyMjPS5uvCgoOlyc+ZhYfnc3Pri4u6X l+VdXv309P77++FDRN4xMt4wMeA9PuA4OeNOTvzq6uyDhN4uL+hrbORWV/bIyN0rLO6TlPfQ0fCo qaxUY9ycocbM2M4yN9UtMO7e4be2xdhBRFaQtjR2opO50Cdwn6zJ2zp/qsHX5KXE2BZnm/L3+Rlq nBhpm0OEruXu9IOuyV2Wuo+3z+sDLFgAAAABYktHRACIBR1IAAAAB3RJTUUH6AsSBCg2Lu0OpQAA AMZJREFUGNNlj912gjAQhKeKSm0kRZvW+NdahQYw2TXA+79aF4537sVcfGd3ZhZ4mpfJNJnNJ4v0 db5M3gC1ynT2rrVIrvVaQeVZvlEfxnwq9aWzzQDWwNaaBNgZnQ5gCeytnQKJMengMQMW1myBg5ET HO0J+P45/wKX81ViivJPxLkKqF0jPW6+QCDiO4L3UUD0LlS+rSk2vh6rOt90HG/ct74dQceFq2Wx LLkZQcXEvTgSURhBJCYx65m7x7/VXYIRumLI+Ad+gRTE5rM7GQAAACV0RVh0ZGF0ZTpjcmVhdGUA MjAyNC0xMS0xOFQwNDo0MDo1NCswMDowMNrCKYMAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjQtMTEt MThUMDQ6NDA6NTQrMDA6MDCrn5E/AAAAKHRFWHRkYXRlOnRpbWVzdGFtcAAyMDI0LTExLTE4VDA0 OjQwOjU0KzAwOjAw/Iqw4AAAAABJRU5ErkJggg==" width="16" height="16"/></svg>
```

Comment:
Can you find an SVG that doesn't have a PNG embedded? If you can't, upload a PNG.

Reply:
I have done my best to find PNG images and compress them to 4KB.

Reply:
Here's an even smaller one, compressed by [TinyPNG](https://tinypng.com).

![Microworkers](https://github.com/user-attachments/assets/8a399f86-f318-4b1a-9740-f6a21f16dfa3)

Reply:
Thanks! I have committed it again!